bin/lhm-kill-queue

```diff
@@ -115,10 +114,10 @@ module Lhm
     def kill_process(process_id)
       puts "killing #{ select_statement(process_id) }"
 
-      if(@confirm)
+      if @confirm
         print "confirm ('y' to confirm): "
 
-        if(gets.strip != 'y')
+        if (gets.strip != 'y')
```

Comment:
You mean on #117?  This is probably due to the way rubocop auto-fixes work.  It has this nasty tendency to fix unrelated style issues, even if they're disabled. :-(  I will put it back and re-run the rubocop test.